fs/operations/operations.go

```diff
@@ -1614,11 +1614,27 @@ func RcatSize(ctx context.Context, fdst fs.Fs, dstFileName string, in io.ReadClo
 	return obj, nil
 }
 
+// CopyURLParseHeader returns a valid filename from a Content-Disposition HTTP string
```

Comment:
In Go a capitalised first letter makes a function externally accessible to the package.

I don't think this one needs to be.

So I'd probably rename it `parseContentDisposition` which seems more accurate.